packages/elements-core/src/components/TryIt/build-request.ts

```diff
@@ -300,3 +310,17 @@ function uriExpand(uri: string, data: Dictionary<string, string>) {
     return data[value] || value;
   });
 }
+
+function getAcceptedMimeTypes(httpOperation: IHttpOperation): string[] {
+  return [
+    ...new Set(
+      httpOperation.responses
+        .map(response =>
+          response === undefined || response.contents === undefined
+            ? []
+            : response.contents.map(contentType => contentType.mediaType),
+        )
+        .reduce((a, b) => [...a, ...b]),
```

Comment:
Currently, tests are failing on this line:

/home/circleci/project/packages/elements-core/src/components/TryIt/build-request.ts:237
            .reduce((a, b) => [...a, ...b])),
             ^

TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>
    
    
I suspect you just need to change it to `.reduce((a, b) => [...a, ...b], [])` to give reduce an initial value to return in the case there's nothing to reduce